src/common/simple_spin.cc

```diff
@@ -18,9 +18,12 @@
 #include <stdint.h>
 #include <pthread.h>
 
-static uint32_t bar = 13;
-static uint32_t *foo = &bar;
- 
+#if (__arm__ || __aarch64__)
```

Comment:
@badone no, `defined()` is better in the sense that we are testing if they are `defined` or not. but C/C++ preprocessor evaluate the undefined macro to 0. so we don't need to change the `src/test/test_arch.cc`, unless we happen to touch them, and want to make them **look** better.